docs/reference/commandline/exec.md

```diff
@@ -39,6 +39,12 @@ The command started using `docker exec` only runs while the container's primary
 process (`PID 1`) is running, and it is not restarted if the container is
 restarted.
 
+> **Note**: COMMAND run using the last `WORKDIR` specified.
```

Comment:
I like this. Removing a bit of jargon. @jmaitrehenry can you please address this?